src/main/webapp/index.jsp

```diff
@@ -136,4 +136,4 @@
             <div style="clear: both;"></div>
         </div>
     </div>
-</t:staticPage>
\ No newline at end of file
+</t:staticPage>
```

Comment:
@jakers821 This change should not be part of the PR, but since this is trivial, I will let it pass. If you wish to contribute more fixes, be careful to not include unrelated changes like this.